assets/js/script.js

```diff
@@ -2,80 +2,76 @@
 
 //Defining function to generate user's password
 
-var generatePassword = function(){
+const generatePassword = function(){
 
   //Defining all possible characters for password and password length in object
-  var passwordChars = {
+  const passwordChars = {
     upperCase:"ABCDEFGHIJKLMNOPQRSTUVWXYZ",
     lowerCase:"abcdefghijklmnopqrstuvwxyz",
     numerics:"1234567890",
     specialChar:"!'`~@#$%^&*()_+-][}{|><?;:=",
     passwordLimit: [8,128]
   }
 
+  const SELECT_LENGTH_PROMPT = "Please choose your password length between 8-128 characters."
```

Comment:
Pulling all constant strings out into a constants block is standard practice.  You can reuse var names and know where all the strings are.  It's also often a practice to pull all the constants for an entire repo into a single constants.js file and import it where they're needed.